snippets5000/LocateProjects/PullRequestSimulationsHelpers/PullRequest.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Text.Json.Serialization;
+using System.Text.Json;
+
+namespace PullRequestSimulations;
+
+public class PullRequest
+{
+    public string Name { get; set; }
+
+    public ChangeItem[] Items { get; set; }
+
+    public ExpectedResult[] ExpectedResults { get; set; }
+
+    public int CountOfEmptyResults { get; set; }
+
+    public PullRequest() { }
```

Comment:
I think you can fix all the warnings with the following changes. You might consider also changing `get; set;` to `get; init;` if you want this struct to be immutable.

```suggestion
    public required string Name { get; set; }

    public required ChangeItem[] Items { get; set; }

    public required ExpectedResult[] ExpectedResults { get; set; }

    public required int CountOfEmptyResults { get; set; }
```

Reply:
Done!